ppr-ui/src/components/collateral/vehicleCollateral/factories/useCollateralValidation.ts

```diff
@@ -39,12 +39,8 @@ export const useCollateralValidation = () => {
   const validateCollateralForm = async currentVehicle => {
     const validationResult = await formValidation.validateForm(currentVehicle)
     errors.value = { ...errors.value, ...validationResult.fieldErrors }
-    if ((currentVehicle.type === 'MH') && (errors.value.manufacturedHomeRegistrationNumber.succeeded)) {
-      errors.value.manufacturedHomeRegistrationNumber =
-        validationResult.recordErrors.serialNumber
-    } else {
-      errors.value.serialNumber = validationResult.recordErrors.serialNumber
-    }
+    errors.value.serialNumber = validationResult.recordErrors.serialNumber
+
```

Comment:
Just to clarify, the serial number field was displaying under the wrong field? 
If so, does the condition need to be moved elsewhere?

Reply:
The bug was mhr number field showing serial number field's error, caused by line 43-44, where serial number error overwrites mhr number error message. So I removed the whole condition statement,  but the test won't pass, so I added back the else statement for the tests to pass.

Reply:
oh we need to keep the line that was in the else statement, is because serial number only appears in record check, so when we are adding only the field error to our error object (line 41), we'll miss out serial number's error; and so we need to explicitly add it.

Reply:
Sounds good, thanks for digging into this a bit more!